src/pages/ANL-confirm.html

```diff
@@ -123,6 +128,7 @@ <h4>What's Next?</h4>
         </p>
         <p>
             UC Davis Analytical Lab <br />
+            <a href="tel:5307520147">530 752-0147</a> <br/>
```

Comment:
@srkirkland I think this is correct for a clickable phone number.

Reply:
@jSylvestre yea that should work